Makefile

```diff
@@ -463,21 +481,101 @@ site/src/api/typesGenerated.ts: scripts/apitypings/main.go $(shell find ./coders
 docs/admin/prometheus.md: scripts/metricsdocgen/main.go scripts/metricsdocgen/metrics
 	go run scripts/metricsdocgen/main.go
 	cd site
-	yarn run format:write ../docs/admin/prometheus.md
+	yarn run format:write:only ../docs/admin/prometheus.md
 
 coderd/apidoc/swagger.json: $(shell find ./scripts/apidocgen -not \( -path './scripts/apidocgen/node_modules' -prune \) -type f) $(wildcard coderd/*.go) $(wildcard codersdk/*.go)
 	./scripts/apidocgen/generate.sh
 	cd site
-	yarn run format:write ../docs/api ../docs/manifest.json ../coderd/apidoc/swagger.json
+	yarn run format:write:only ../docs/api ../docs/manifest.json ../coderd/apidoc/swagger.json
 
 update-golden-files: cli/testdata/.gen-golden
 .PHONY: update-golden-files
 
 cli/testdata/.gen-golden: $(wildcard cli/testdata/*.golden) $(GO_SRC_FILES)
-
 	go test ./cli -run=TestCommandHelp -update
 	touch "$@"
 
+# Ensure there's a .prettierrc.yaml project-wide when prettier is run outside
+# the site package.
+site/.prettierrc.yaml: .prettierrc.yaml Makefile
```

Comment:
Depending on Makefile doesn't seem very necessary and complicates some of the rules below. We also don't do this elsewhere in the Makefile so for consistency I think we should remove it as a dependency. If people are working on the Makefile and need to test changes, they can use `--assume-old` or similar flags to force make to regenerate files.

Reply:
Ok, that's sensible. I ran into this myself and didn't know about `--assume-old`.